### ISSUE TITLE
[BUG] Something happened and the Taiga has captured the error to be able to work on it.

### ISSUE THREAD

**Issue by GraeMSaunders** [taigaio/taiga-docker#23]
**Describe the bug**
I have followed the instructions for launching a new instance (https://resources.taiga.io/30min-setup/). I have a VM setup in Azure that I am doing this testing on. Due to firewall restrictions I need to run Nginx to proxy port 80 to localhost:9000. This seems to be a very common problem so I am not sure if it is just a simple config item I am missing. I will detail my steps further below.

**How can we reproduce the behavior**
1. Start fresh Ubuntu 20.04 image in Azure
2. Install docker and docker-compose as per https://docs.docker.com/engine/install/
3. Git clone and checkout stable.
4. I configure docker-compose.yml with values of my ip for 
- TAIGA_SITES_DOMAIN: "a.b.c.d:9000"
- TAIGA_URL: "http://a.b.c.d:9000"
- TAIGA_WEBSOCKETS_URL: "ws://a.b.c.d:9000"
5. `./launch-all.sh`
6. `./taiga-manage.sh createsuperuser`
7. At this point I get the below error so I cancel the superuser creation
```
You have 257 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, attachments, auth, bitbucket, contact, contenttypes, custom_attributes, djmail, easy_thumbnails, epics, external_apps, feedback, github, gitlab, gogs, history, issues, likes, milestones, notifications, projects, references, sessions, settings, tasks, telemetry, timeline, users, userstorage, userstories, votes, webhooks, wiki.
Run 'python manage.py migrate' to apply them.
```
8. I run `./taiga-manage.sh migrate` as per the message
9. I run `./taiga-manage.sh createsuperuser` and works as expected
10. I install nginx and configure both the default path and events path as per instructions.
11. I open a browser and broswe to http://a.b.c.d 
12. Things appear to load for a while then I get the "Something happened and the Taiga has captured the error to be able to work on it." message.

**Comment by GraeMSaunders** [comment]
Ok I have solved this. What isnt clear in any of the documentation is that the values in the docker-compose.yml need to be accessible from the client. Once I update the below to the port I was proxying from it all started working.
TAIGA_SITES_DOMAIN: "a.b.c.d:80"
TAIGA_URL: "http://a.b.c.d:80"
TAIGA_WEBSOCKETS_URL: "ws://a.b.c.d:80"

Is there anything else I should be concerned about not working with this setup? Is this something that can be updated in the server setup instructions?

**Comment by yamila-moreno** [comment]
Ey @GraeMSaunders glad you fixed it.

Regarding the issue with the migrations, I'll check it.

Regarding the configuration, a couple of things:
- TAIGA_URL is the url where taiga is served, so you can write "https://taiga.company.com" or "http://taiga.company.com" if you don't have TSL configured.
- TAIGA_WEBSOCKETS_URL behaves similarly: wss://taiga.company.com or ws://taiga.company.com
- TAIGA_SITES_SCHEME + TAIGA_SITES_DOMAIN should be like TAIGA_URL, so "http" and "taiga.mycompany.com"

You mention that we can update the instructions, and we'd be happy to do so. I'd like to know better what would be useful for you in the documentation in order to configure this properly. 

Cheers!

**Comment by nurulakbartan** [comment]
i have same error

in docker compose yml

in part

TAIGA_SITES_DOMAIN: "localhost:9000"
TAIGA_SITES_SCHEME: "http"

i changed into

TAIGA_SITES_DOMAIN: "pm.tjakrabirawa.id"
TAIGA_SITES_SCHEME: "https"

  TAIGA_URL: "http://localhost:9000"
  TAIGA_WEBSOCKETS_URL: "ws://localhost:9000"
i changed into

  TAIGA_URL: "https://pm.tjakrabirawa.id"
  TAIGA_WEBSOCKETS_URL: "wss://pm.tjakrabirawa.id"

**Comment by mm86133** [comment]
It is clearly a bug.

Same error:
TAIGA_SITES_DOMAIN: "localhost:9000"
TAIGA_SITES_SCHEME: "http"


to

TAIGA_SITES_DOMAIN: "10.101.3.1:9000"
TAIGA_SITES_SCHEME: "http"


Fresh installation in 95% according to manual (I've changed the site domain as it wasn't mentioned in documentation, I am getting also same error when copying the manual at 100%).


Unbutu 18.04, Docker v 20.10.6

**Comment by gponcon** [comment]
I also have this error and updating the domain name doesn't change anything. I've tried many times to delete all the volumes and then reinstall, trying different versions, and it's confusing to always have the message "Something happened and the Taiga has captured the error to be able to work on it." with no other information and no error message in the container logs when the error occurs. 

There are warnings `WARNING: '/var/lib/rabbitmq/.erlang.cookie' was populated from '$RABBITMQ_ERLANG_COOKIE', which will no longer happen in 3.9 and later!` and `WARNING: 'docker-entrypoint.sh' generated/modified the RabbitMQ configuration file, which will no longer happen in 3.9 and later!` which shouldn't be a problem as we're still using version 3.8...

This error also appears: `[error] <0.733.0> closing AMQP connection <0.733.0> (172.23.0.8:56330 -> 172.23.0.5:5672): missed heartbeats from client, timeout: 60s` but it's impossible to know if this is the cause and how to resolve the problem.

In short, this bug has not been resolved.
